octopusdeploy_framework/resource_docker_container_registry.go

```diff
@@ -0,0 +1,182 @@
+package octopusdeploy_framework
+
+import (
+	"context"
+	"fmt"
+	"github.com/OctopusDeploy/go-octopusdeploy/v2/pkg/core"
+	"github.com/OctopusDeploy/go-octopusdeploy/v2/pkg/feeds"
+	"github.com/OctopusDeploy/terraform-provider-octopusdeploy/octopusdeploy_framework/schemas"
+	"github.com/OctopusDeploy/terraform-provider-octopusdeploy/octopusdeploy_framework/util"
+	"github.com/hashicorp/terraform-plugin-framework/attr"
+	"github.com/hashicorp/terraform-plugin-framework/resource"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+	"github.com/hashicorp/terraform-plugin-log/tflog"
+)
+
+type dockerContainerRegistryFeedTypeResource struct {
+	*Config
+}
+
+func NewDockerContainerRegistryFeedResource() resource.Resource {
+	return &dockerContainerRegistryFeedTypeResource{}
+}
+
+func (r *dockerContainerRegistryFeedTypeResource) Metadata(ctx context.Context, req resource.MetadataRequest, resp *resource.MetadataResponse) {
+	resp.TypeName = util.GetTypeName("docker_container_registry")
+}
+
+func (r *dockerContainerRegistryFeedTypeResource) Schema(ctx context.Context, req resource.SchemaRequest, resp *resource.SchemaResponse) {
+	resp.Schema = schema.Schema{
```

Comment:
This needs a `Description` field with the description from the old schema file.

Reply:
I've fixed the descriptions across all the feeds and other resources I've migrated [here](https://github.com/OctopusDeployLabs/terraform-provider-octopusdeploy/pull/702/commits/124b893cd90260c9ba7612627cfe5b4296a63466).